### ISSUE TITLE
`balena os initialize` no longer initialises an image

### ISSUE THREAD

**Issue by hedss** [balena-io/balena-cli#1428]
- **balena CLI version:** 11.9.5
- **Operating system version:** macOS 10.14.5
- **32/64 bit OS and processor:** 64-bit Intel processor
- **Install method:** npm
- **If npm install, Node.js and npm version:** Node v10.16.0 and npm v6.9.0

Running the command results in balenaCLI returning the help text (as in an invalid command has been run):
```
$ DEBUG=1 balena os initialize balena-cloud-freee-raspberrypi3-2.38.0+rev1-dev-v9.15.7.img --type raspberrypi3 --drive /dev/disk3 -y
[debug] original argv0="node" argv=[/Users/heds/.nvm/versions/node/v10.16.0/bin/node,/Users/heds/.nvm/versions/node/v10.16.0/bin/balena,os,initialize,balena-cloud-freee-raspberrypi3-2.38.0+rev1-dev-v9.15.7.img,--type,raspberrypi3,--drive,/dev/disk3,-y] length=10
Initializing device

Note: Initializing the device may ask for administrative permissions
because we need to access the raw devices directly.
Going to erase /dev/disk3.
If asked please type your computer password to continue
[debug] original argv0="/Users/heds/.nvm/versions/node/v10.16.0/bin/node" argv=[/Users/heds/.nvm/versions/node/v10.16.0/bin/node,/Users/heds/.nvm/versions/node/v10.16.0/bin/balena,internal,sudo,internal osinit balena-cloud-freee-raspberrypi3-2.38.0+rev1-dev-v9.15.7.img raspberrypi3 {"drive":"/dev/disk3"}] length=5
Usage: balena [COMMAND] [OPTIONS]

If you need help, or just want to say hi, don't hesitate in reaching out
through our discussion and support forums at https://forums.balena.io

For bug reports or feature requests, have a look at the GitHub issues or
create a new one at: https://github.com/balena-io/balena-cli/issues/

Primary commands:

    help [command...]                       show help
    login                                   login to balena
    push <applicationOrDevice>              Start a remote build on the balena cloud build servers or a local mode device
    logs <uuidOrDevice>                     show device logs
    ssh <applicationOrDevice> [serviceName] SSH into the host or application container of a device
    apps                                    list all applications
    app <name>                              list a single application
    devices                                 list all devices
    device <uuid>                           list a single device
    tunnel <deviceOrApplication>            Tunnel local ports to your balenaOS device
    preload <image>                         preload an app on a disk image (or Edison zip archive)
    build [source]                          Build a single image or a multicontainer project locally
    deploy <appName> [image]                Deploy a single image or a multicontainer project to a balena application
    join [deviceIp]                         Promote a local device running balenaOS to join an application on a balena server
    leave [deviceIp]                        Detach a local device from its balena application
    scan                                    Scan for balenaOS devices in your local network

Run `balena help --verbose` to list additional commands

Global Options:

    --help, -h
    --version, -v
You can safely remove /dev/disk3 now
```
I believe that the JSON data passed as a string is probably incorrectly formatted, hence the issue (a quick poke seems to confirm this), but there could potentially be other issues with the way `internal sudo` is run.

**Comment by emg-rexnord** [comment]
I'm experiencing the same issue.

**balena CLI version:** 11.12.0
**Operating system version:** Ubuntu 19.04 kernel 5.0.0-29-generic
**32/64 bit OS and processor:** 64-bit Intel processor
**Install method:** npm
**If npm install, Node.js and npm version:** Node v10.16.3 and npm v6.9.0

Any workarounds? Is there a version of the CLI that I can roll back to where this functionality works?

**Comment by pdcastro** [comment]
This issue was fixed in CLI release 11.14.5 ([releases](https://github.com/balena-io/balena-cli/releases)).
